### PR TITLE
[AWN-32688] Accordion Component

### PR DESCRIPTION
**Pull request by rtkaaho**
## Description:
- Adding accordion component as an orchestration of collapse components

## Testing:
- Unit Testing Implemented

 ## Screenshots:
Screen Shot 2019-12-10 at 10 44 12 AM

### CODE REVIEW

File: src/components/accordion/__tests__/Accordion.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  it('renders all children', () => {
+    const wrapper = shallow(
+      <Accordion>
+        <Accordion.Item itemKey={'1'}>Accordion Item 1</Accordion.Item>
+        <Accordion.Item itemKey={'2'}>Accordion Item 2</Accordion.Item>
+        <Accordion.Item itemKey={'3'}>Accordion Item 3</Accordion.Item>
+      </Accordion>
+    );
+
+    expect(wrapper.exists('Accordion__Container')).toBe(true);
```
Comment (rtktpeng) [inline src/components/accordion/__tests__/Accordion.spec.tsx]: I didn't know jest is converting dots into double underscores. Cool!

File: src/components/accordion/__tests__/Accordion.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+import {
+  mount, shallow
+} from 'enzyme';
+
+import {
+  Accordion
+} from '../Accordion';
+
+import 'jest-styled-components';
```
Comment (rtktpeng) [inline src/components/accordion/__tests__/Accordion.spec.tsx]: Does this need to be imported here? I see it being imported in setupTests already.

File: src/components/accordion/__tests__/Accordion.spec.tsx (hunk 1 above)
Comment (rtkaaho) [reply src/components/accordion/__tests__/Accordion.spec.tsx]: This is actually styled-components :) it takes the `styled-component` in this case `Container` and prepends the component that implements it `Accordion` to create that kind of jsx name.

File: src/components/accordion/__tests__/Accordion.spec.tsx (hunk 2 above)
Comment (rtkaaho) [reply src/components/accordion/__tests__/Accordion.spec.tsx]: good catch will remove that now

**Pull request merged**